_data/changelogs/component-card.yml

```diff
@@ -2,6 +2,14 @@ title: Card
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added the `$theme-card-font-family` to the card container typeset mixin.
+    summaryAdditional: Now, cards respect the `$theme-card-font-family` theme setting.
```

Comment:
**suggestion:** Recommend we re-write this to focus on the main thing the users will care about. 

```suggestion
    summary: Fixed a bug that caused the component to ignore the `$theme-card-font-family` setting.
```